### DISCUSSION TITLE
Where to send the additional proof of academic status?

### DISCUSSION THREAD

**Discussion by Vicentesan** [community/community#106526] [GitHub Education] [answered]
### Hi everyone,

Hello everyone, I could really use some help figuring out where I should send this additional proof. I'm feeling a bit lost at the moment.

![image](https://github.com/community/community/assets/143416369/6215b6f2-333c-4265-8bdb-7c0a7f600bbf)

**Comment by Anixton** [comment]
Hi @Vicentesan , first of all, let me welcome you to GitHub Community :)
I assume you need to prove your academic status to apply for the GitHub Student Developer pack. If that's the case, continue reading on.
With regards to your application for GitHub Student Developer pack, you can consider submitting documents such as the following:

Picture of your school ID with current enrollment date

Academic transcript

School affiliation or enrollment verification letter

Class schedule

If all fails, please submit a ticket to Github Education Support over here
Hope this helps!
Reference:
https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-for-your-schoolwork/apply-for-a-student-developer-pack

Please mark this comment as answer :)

**Reply by Vicentesan** [reply]
Yes, I understand that I need to send these additional documents, but where should I send them? Because when I select the school email again, that request appears and it blocks the button to continue with the request... I had already made the request by sending my school ID, but it was rejected with this warning

**Reply by Anixton** [reply]
https://support.github.com/contact/education

Check here

**Reply by Vicentesan** [reply]
If I select any option that includes "Github Education" I cannot continue writing the ticket and this modal opens

![image](https://github.com/community/community/assets/143416369/7c23bb09-dd64-49f1-aaf5-b2362c20105f)
![image](https://github.com/community/community/assets/143416369/e3c1d2e7-5fb0-4cae-a2cb-b717712cc03d)

**Reply by Anixton** [reply]
Well i didn't know this. You may try different problem path for support after that they may direct your mail to somebody else at the mean time you can try applying again sometimes that works too.

**Reply by Vicentesan** [reply]
I'll try it! Thank you for you help

**Comment by samus-aran** [comment]
Hi @Vicentesan 

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

You should be asked after the "Continue" button to either capture your evidence in a live photo or an upload (depending if you're on campus).

**Reply by Vicentesan** [reply]
Hi @samus-aran

This error appeared after my first and only attempt to apply for the GitHub Student Developer Pack. After receiving this email, I did everything that was written and logged out of all accounts to log in again. After this entire process, I tried to resend the request and received the error that I previously showed.

On the first attempt I had already sent my school ID

![image](https://github.com/community/community/assets/143416369/97ff6146-858a-4beb-b9bc-76536683dc34)

**Reply by samus-aran** [reply, marked as answer]
Okay dokay. So I need you to confirm that you have 2FA enabled. That is the first step.

The message "We require students...." isn't an error but just letting you know that when you click continue it will ask you to upload/take a photo of additional evidence.

The problem I see in your screenshots is that Continue button is disabled. This would indicate a couple of things:

- You haven't allowed location access on your browser which is required
- You need to refresh your browser cache

Try these first and if you are unable to proceed beyond the "Continue" page let us know. 👍

**Reply by Vicentesan** [reply]
I just sent a second attempt, thank you very much for your help! Now just wait for approval

**Reply by Vicentesan** [reply]
@samus-aran , thank you for your help!

![image](https://github.com/community/community/assets/143416369/bf3068bf-8188-43af-9b9c-9f4b5312fa5e)
